src/ng/parse.js

```diff
@@ -1698,12 +1698,14 @@ function $ParseProvider() {
   var cacheExpensive = createMap();
 
   this.$get = ['$filter', '$sniffer', function($filter, $sniffer) {
+    var csp = $sniffer.csp && !$sniffer.unsafeEval;
```

Comment:
Oh I see what you're saying, we don't actually have a check for unsafe-inline anymore, I guess that disappeared. So yeah, `csp` is just an alias for this at the moment.